Ginger/GingerCoreNET/ActionsLib/ActionsConversion/eConversionSaveStatus.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Amdocs.Ginger.CoreNET
+{
+    /// <summary>
+    /// This enum is used to hold the save status for the converted businessflows
+    /// </summary>
+    public enum eConversionSaveStatus
```

Comment:
no need seperate class for this enum, you can add in BusinessFlowToConvert.cs outside class